src/rgw/driver/rados/rgw_reshard.cc

```diff
@@ -957,7 +1048,7 @@ int RGWBucketReshard::execute(int num_shards,
   auto unlock = make_scope_guard([this] { reshard_lock.unlock(); });
 
   if (reshard_log) {
-    ret = reshard_log->update(dpp, bucket_info, y);
+    ret = reshard_log->update(dpp, bucket_info, initiator, y);
     if (ret < 0) {
       return ret;
     }
```

Comment:
@cbodley : I'm trying to think of why when having decided to execute a reshard we would update the reshard queue (log). In testing I've commented out this logic and I've seen no adverse impact, although maybe I haven't brushed up against the reason it's there. Do you have any thoughts?